.gitmodules

```diff
@@ -35,3 +35,6 @@
 	path = tpls/qpp
 	url = https://github.com/softwareQinc/qpp.git
 	shallow = true
+[submodule "tpls/cpr"]
```

Comment:
It's odd to add this as a submodule if we are not actually using the submodule to build the code - I think this is just confusing, because changing the submodule commit will not change the code.

Reply:
I can add the changes to cmake to this PR. I had thought it made more sense to make it 2 steps.